docs/index.rst

```diff
@@ -104,6 +104,22 @@ A Glance:
     u'The study shows that 93% of people ...'
 
 
+Newspaper can directly parse a string:
+
+.. code-block:: pycon
+
+    >>> from newspaper import Article
+
+    >>> a = Article('http://example.com/fake')  # Pass any url
+
+    >>> my_html = u'<html><title> ...'
+
+    >>> a.set_html(my_html)  
```

Comment:
I think it would be helpful to comment this line as well. 

> This is where you will pass in your file to override the URL you passed in the previous step.